### DISCUSSION TITLE
Cris Motinha: a brasileira escolhida como 'Inovadora' para o MongoDB (EUA)

### DISCUSSION THREAD

**Discussion by leandroruel** [frontendbr/forum#552] [Discussão]
Cristina Motinha, ou Cris Motinha, começou a "programar" aos 10 anos de idade. Doze anos depois, especificamente aos 22 anos, Cris é uma das brasileiras selecionadas como "Female Innovators" pela MongoDB World 2017, que acontece em Chicago (EUA), nos mês de junho. #orgulho #vaiBrasil

leiam a matéria completa: https://www.tecmundo.com.br/programacao/116096-cris-motinha-brasileira-escolhida-inovadora-mongodb-eua.htm?utm_source=tecmundo.com.br&utm_medium=home&utm_campaign=tv

ps: na moral, se alguém conhecer ela, convidem ela para participar do nosso fórum e falar sobre isso ai.

**Comment by mksanti** [comment]
Eu li essa matéria hoje pela manhã e teve uma coisa que me fez sentir com retardo metal.

> Em janeiro de 2016 foi quando comecei oficialmente minha jornada na programação. Me inscrevi no curso HTML/CSS do Codecademy e comecei a fazer todas as tarefas, uma atrás da outra, como um vício. Depois, foi a vez do JavaScript, e depois mais JS pelo W3Schools".

> Encontrei o Django Girls [...] Fiquei encantada com Django e com Python. Tentava associar tudo ao que eu já conhecia, e montei esse quebra cabeça louco que misturava chaves com 'def' e por aí vai.[...] Agora já tem uns 15 meses desde que comecei essa jornada. Já fiz mil coisas com JS, com Python, com C, com .NET, enfim... Vi mil coisas em pouquíssimo tempo".

**Comment by leandroruel** [comment]
tem casos que não ficam famosos, mas tem muita gente que ocorre desse jeito tbm

**Comment by melissarib** [comment]
Eu tenho o contato da Cris por causa dos grupos Django Girls e Pyladies no telegram, vou avisá-la sobre o fórum.

**Comment by crismotinha** [comment]
oi :3
Agora tô no trabalho, fica ruim de responder, mas assim que der eu volto aqui \o\ se tiver algum grupo no telegram fica até mais fácil ^^

**Comment by leandroruel** [comment]
Eae cris... welcome

**Comment by WandersonAlves** [comment]
Bem que podia ter mesmo um grupo no telegram, fica a ideia hm

Edit: (esqueci de escrever rs) acho FODA esse tipo de coisa, no momento estava passando por uma reflexão, meio desmotivado para aprender algo novo e basicamente continuar com o que trabalho hoje, mas lendo essa materia, acendeu a chama do aprendizado novamente. Obrigado Cris!

**Comment by fabiomirandaa** [comment]
Arrasa lá @crismotinha

**Comment by felipefialho** [comment]
Sensacional @crismotinha, bem vinda a comunidade e aguardamos seu depoimento 😄 

ps. Problema de grupos de Telegram e etc, é a não escalabilidade do mesmo. Seu depoimento escrito aqui ficará indexado nos buscadores e na internet, hahaha.

**Comment by leandroruel** [comment]
é não deixe de contar pra gente em @crismotinha 

w/ :heart:

**Comment by suissa** [comment]
@crismotinha https://t.me/frontendbrasil
